core/src/layers/madsim.rs

```diff
@@ -134,7 +134,7 @@ impl LayeredAccessor for MadsimAccessor {
     fn metadata(&self) -> AccessorInfo {
         let mut info = AccessorInfo::default();
         info.set_name("madsim");
-        info.set_capabilities(AccessorCapability::Read | AccessorCapability::Write);
```

Comment:
So now the capability can be set in two ways?

`Read | Write` or `Capability { read: true, write: true, ..Default::default()}`?

Reply:
> So now the capability can be set in two ways?

No, Only via `Capability { .. }`

Reply:
You may need to update this line.

Reply:
madsim layer is borken for it's `cfg` requirement. I will fix it in another issue.

Reply:
Tracked at https://github.com/apache/incubator-opendal/issues/2058